specification/edgemarketplace/resource-manager/readme.typescript.md

```diff
@@ -0,0 +1,13 @@
+## TypeScript
+
+These settings apply only when `--typescript` is specified on the command line.
+Please also specify `--typescript-sdks-folder=<path to root folder of your azure-sdk-for-js clone>`.
+
+```yaml $(typescript)
+typescript:
+  azure-arm: true
+  package-name: "edgemarketplace"
```

Comment:
pleae use @azure/arm- as the package prefix,  and review the name with SDK arch board.

Reply:
@qiaozha , Updated name. We are planning SDK release after a few months. Will do the review with SDK board.